src/electionguard-ui/ElectionGuard.UI/ViewModels/CreateKeyCeremonyAdminViewModel.cs

```diff
@@ -40,7 +40,7 @@ public async Task CreateKeyCeremony()
             return;
         }
 
-        var keyCeremony = new KeyCeremonyRecord(KeyCeremonyName, Quorum, NumberOfGuardians, this.UserName!);
+        var keyCeremony = new KeyCeremonyRecord(KeyCeremonyName, NumberOfGuardians, Quorum, this.UserName!);

```

Comment:
just since we have had this issue in the past:

```suggestion
        var keyCeremony = new KeyCeremonyRecord(name: KeyCeremonyName,
                                                numberOfGuardians: NumberOfGuardians,
                                                quorum: Quorum,
                                                admin: this.UserName!);
```